css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These should be in the opposite order, take another look at the instructions!

Reply:
Corrected, thanks for your review.
![image](https://user-images.githubusercontent.com/76827487/113949671-a7d3ec80-97c4-11eb-8a75-e7380296afd7.png)